tests/integration/docusaurus/validation/checkpoints/how_to_validate_data_with_a_python_configured_in_memory_checkpoint.py

```diff
@@ -0,0 +1,121 @@
+# Required imports for this script's purpose:
+# <snippet>
+import great_expectations as ge
+from great_expectations.checkpoint import Checkpoint
+
+# </snippet>
+
+# Imports used for testing purposes (and can be left out of typical scripts):
+from great_expectations.core.expectation_validation_result import (
+    ExpectationSuiteValidationResult,
+)
+from great_expectations.core.run_identifier import RunIdentifier
+from great_expectations.data_context.types.base import CheckpointConfig
+from great_expectations.data_context.types.resource_identifiers import (
+    ValidationResultIdentifier,
+)
+
+# Import and setup for working with YAML strings:
+# <snippet>
+from ruamel import yaml
+
+yaml = yaml.YAML(typ="safe")
+# </snippet>
+
+# Initialize your data context.
+# <snippet>
+context = ge.get_context()
+# </snippet>
+
+# Add datasource for all tests
+datasource_yaml = """
+name: taxi_datasource
+class_name: Datasource
+module_name: great_expectations.datasource
+execution_engine:
+  module_name: great_expectations.execution_engine
+  class_name: PandasExecutionEngine
+data_connectors:
+  default_inferred_data_connector_name:
+    class_name: InferredAssetFilesystemDataConnector
+    base_directory: ../data/
+    default_regex:
+      group_names:
+        - data_asset_name
+      pattern: (.*)\\.csv
+  default_runtime_data_connector_name:
+    class_name: RuntimeDataConnector
+    batch_identifiers:
+      - default_identifier_name
+"""
+context.test_yaml_config(datasource_yaml)
+context.add_datasource(**yaml.load(datasource_yaml))
+assert [ds["name"] for ds in context.list_datasources()] == ["taxi_datasource"]
+# Add Expectation Suite for use in Checkpoint config
+context.create_expectation_suite("my_expectation_suite")
+
+# Define your checkpoint's configuration.
+# NOTE: Because we are directly using the Checkpoint class, we do not need to
+# specify the parameters `module_name` and `class_name`.
+# <snippet>
+my_checkpoint_name = "in_memory_checkpoint"
+python_config = {
+    "name": my_checkpoint_name,
+    "config_version": 1,
+    "run_name_template": "%Y%m%d-%H%M%S-my-run-name-template",
+    "action_list": [
+        {
+            "name": "store_validation_result",
+            "action": {"class_name": "StoreValidationResultAction"},
+        },
+        {
+            "name": "store_evaluation_params",
+            "action": {"class_name": "StoreEvaluationParametersAction"},
+        },
+        {
+            "name": "update_data_docs",
+            "action": {"class_name": "UpdateDataDocsAction", "site_names": []},
+        },
+    ],
+    "validations": [
+        {
+            "batch_request": {
+                "datasource_name": "taxi_datasource",
+                "data_connector_name": "default_inferred_data_connector_name",
+                "data_asset_name": "yellow_tripdata_sample_2019-02.csv",
```

Comment:
I think this is a typo here: changing `data_asset_name` to `yellow_tripdata_sample_2019-02` should make tests pass

Reply:
I've updated it to use the same name that was used in the YAML version of the config.